lib/services/metadata_provider.dart

```diff
@@ -84,23 +84,20 @@ final AutoDisposeFutureProviderFamily<MetadataProvider?, MetadataRequest>
       metadataProviderLogger
           .fine("Got offline metadata for '${request.item.name}'");
       var profile = downloadItem.fileTranscodingProfile;
-      var codec = profile?.codec == FinampTranscodingCodec.original
-          ? downloadItem.baseItem?.mediaSources?.first.container
-          : profile?.codec.name ?? FinampTranscodingCodec.original.name;
-      
-      var mediaStream = downloadItem.baseItem!.mediaStreams?.map((mediaStream) {
-        // if we transcoded the file, we need to update the original bitrate
-        // the bitrate of the MediaSource(Info) includes all streams, so we prefer the media stream bitrate and hence update it here
-        if (["Audio", "Default"].contains(mediaStream.type)) {
-          //!!! the bitrate for the original codec is just a placeholder
-          // keep the original bitrate from the media stream instead
-          if (profile?.codec != FinampTranscodingCodec.original && profile?.stereoBitrate != null) {
-            mediaStream.bitRate = profile?.stereoBitrate;
-          }
-        }
-        return mediaStream;
-      }).toList() ?? [];
-          
+      // We could explicitly get a mediaSource of type Default, but just grabbing
+      // the first seems to generally work?
```

Comment:
Not sure if that's a good idea. I was thinking about supporting stuff like music videos, where the bitrate should show the audio bitrate, and not the video bitrate (but the video would most likely be the first stream).  
Preferring type `Audio`, then `Default`, and then falling back to the first stream would be the best approach imo

Reply:
Those aren't both possible values.  MediaSources can have types "Default" "Grouping" "Placeholder", while Mediastreams can have types "Audio" "Video" "Subtitle" "EmbeddedImage".

Reply:
Interesting. In that case maybe grab the first source, but the `Audio` stream?

Reply:
Yeah, that's basically how I've got it set up now.